### DISCUSSION TITLE
Please improve the optimization

### DISCUSSION THREAD

**Discussion by Likver** [ppy/osu#13803] [Ideas] [answered]
Actually, I don't know how to program, but I think the game is poorly optimized, I have a Sony Xperia L3 and I am surprised that the game goes to less than 20fps even when configuring it, in addition to that I have fps drop of up to 4 fps, so please fix that.

**Comment by frenzibyte** [comment, marked as answer]
Optimization is still yet in progress, and mobile-specific issues are not a priority at the current time.
